zh_cn.json

```
@@ -0,0 +1,139 @@
+{
+  "fluid_type.brewinandchewin.honey_type": "蜂蜜",
+  "fluid_type.brewinandchewin.beer_type": "啤酒",
+  "fluid_type.brewinandchewin.vodka_type": "伏特加",
+  "fluid_type.brewinandchewin.mead_type": "蜂蜜酒",
+  "fluid_type.brewinandchewin.egg_grog_type": "鸡蛋格罗格",
+  "fluid_type.brewinandchewin.strongroot_ale_type": "强根麦芽酒",
+  "fluid_type.brewinandchewin.rice_wine_type": "米酒",
+  "fluid_type.brewinandchewin.glittering_grenadine_type": "发光果浆",
+  "fluid_type.brewinandchewin.steel_toe_stout_type": "钢趾黑啤",
+  "fluid_type.brewinandchewin.dread_nog_type": "惧怖蛋奶酒",
```

```suggestion
  "fluid_type.brewinandchewin.dread_nog_type": "畏怖蛋奶酒",
```
Dread Nog疑似是无畏（Dreadnought）的谐音，去除掉原词中无（Nought）的部分后，剩下的Dread的取字里带上“畏”可能会更好一点。